content/cpp/concepts/math-functions/terms/tanh/tanh.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'tanh()'
+Description: 'Returns the hyperbolic tangent of a double-type value.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`Math.tanh()`** function returns the hyperbolic tangent of an `double`-type argument given in radians.
+
+## Syntax
+
+```pseudo
+std::tanh(angle)
+```
+
+The `angle` is measured in radians and the return type is a `double`, `float`, or `long double`.
+
+The mathematical formula used in `tanh()` looks like this:
+
+```tex
+tanh(x) = (e<sup>x</sup> - e<sup>-x</sup>) / (e<sup>x</sup> + e<sup>-x</sup>)
+```
```

Comment:
@tanishq-singh-2301 Unfortunately, we aren't able to combine \```tex code with GH-flavored HTML (`<sup></sup>`). We have two options:

- Remove the \```tex backticks and keep `<sup></sup>`. (recommended)
- Remove `<sup></sup>` and keep \``` tex code block.
  - Again, this is not likely to render correctly at this time.